minecraftServers/Controllers/ServersController.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using minecraftServers.Repositories;
+using minecraftServers.Servers;
+using System.Text.RegularExpressions;
+
+namespace minecraftServers.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+public class ServersController : ControllerBase
+{
+    private readonly IServersRepository _serversRep;
+    private readonly List<ServerList> _servers = new();
+    public ServersController(IServersRepository serverRep)
+    {
+        _serversRep = serverRep;
+    }
+
+    [HttpGet]
+    public ActionResult<List<ServerList>> GetAll()
+    {
+        return Ok(_serversRep.GetAll());
+    }
+
+    [HttpPost("/api/servers/add")]
+    public ActionResult<ServerList> Add(CustomServer server)
+    { 
+        ServerList? servers = _serversRep.GetName(server.Name);
+        ServerList? serversIp = _serversRep.GetIp(server.Ip);
```

Comment:
When you working with your domain model it's important to choose or create a unique property. In DB field it's called `Primary key`. Choose yours and then it will be easier to work with your model